webapp/src/Service/ImportExportService.php

```diff
@@ -850,9 +850,11 @@ public function importAccountsJson(array $data, ?string &$message = null, ?array
             switch ($account['type']) {
                 case 'admin':
                     $roles[] = $adminRole;
-                    break;
+                case 'jury':
                 case 'judge':
-                    $roles[]  = $juryRole;
+		    if ($account['type'] === 'admin') {
```

Comment:
The formatting is a bit wrong. And maybe we should add a comment instead of the removed break as to why we do this?

Reply:
its also the wrong if statement.